homeassistant/components/mqtt/light/schema_json.py

```diff
@@ -108,13 +110,33 @@ def valid_color_configuration(config: ConfigType) -> ConfigType:
     return config
 
 
+def generate_brightness_scale_map(
```

Comment:
We should probably cache this

Reply:
It could be cached, but would need to copy for each instance, as it would be modified on some writes

Reply:
You could use lru_cache, Example:

https://github.com/home-assistant/core/blob/48f8cec84bc3a7a901530632e3a4d75dc7c9d969/homeassistant/components/mqtt/client.py#L818-L826

Reply:
It is possible to invalidate the cache when it is needed.